frameworks/PHP/fat-free/index.php

```diff
@@ -154,6 +156,7 @@ function ($f3, $params) {
 
 $f3->route(array(
     'GET /update-raw',
+    'GET /update-raw/',
     'GET /update-raw/@queries'
```

Comment:
```suggestion
    'GET /update-raw/',
    'GET /update-raw/@queries'
```
The config is using `/update-raw/`, the same for the rest.

Reply:
This is just my guess since fat free doesn't auto append a forward slash to their URLs. The https://tfb-status.techempower.com/unzip/results.2020-04-20-07-27-56-665.zip/results/20200415180849/fat-free/update/verification.txt file just shows no route, but it's defined in the benchmark_config.json and the multiple db queries and the update test keep failing on the citrine server for some reason, but not in my local codebase in the vagrant box.